Bottom: 298fd670441cc02b98cc7fab38b55254032cf4c6
Top:    ab31537b275d4dd0c6a20822b458338f6f5b6bd2
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-18 16:06:44 +0000

Refresh of fix-a-few-random-places-where

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 3d1b6cc..a27a6b3 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -1692,6 +1692,8 @@ fill_board_outline_holes (hidGC gc, const BoxType *drawn_area)
   struct outline_info info;
 
   ensure_board_outline ();
+  if (PCB->Data->outline == NULL)
+    return;
 
   memset (&polygon, 0, sizeof (polygon));
   polygon.Clipped = PCB->Data->outline;
